protos/mocap/mocap.proto

```diff
@@ -0,0 +1,131 @@
+    uint32 reset_counter = 9; // Estimate reset counter.
+}
+
+
+message AttPosMocapRequest {
+    AttPosMocap att_pos_mocap = 1;
+}
+message AttPosMocapResponse {
+  MocapResult mocap_result = 1;
+}
+
+// Motion capture attitude and position
+message AttPosMocap {
+    uint64 time_usec = 1; // Position frame timestamp UNIX Epoch time (0 to use Backend timestamp)
+    repeated float q = 2; // Attitude quaternion (w, x, y, z order, zero-rotation is 1, 0, 0, 0)
+    float x = 3; // X position (NED)
+    float y = 4; // Y position (NED)
+    float z = 5; // Z position (NED)
+    repeated float covariance = 6; // Row-major representation of pose 6x6 cross-covariance matrix upper right triangle. Set first to NaN if unknown.
+}
+
+
+message OdometryRequest {
+    Odometry odometry = 1;
+}
+message OdometryResponse {
+  MocapResult mocap_result = 1;
+}
+
+// Odometry message to communicate odometry information with an external interface.
+message Odometry {
+    // Mavlink frame id
+    enum MavFrame {
+      MAV_FRAME_GLOBAL = 0; // Global (WGS84) coordinate frame + MSL altitude.
+      MAV_FRAME_LOCAL_NED = 1; // Local coordinate frame, Z-down (x: north, y: east, z: down).
+      MAV_FRAME_MISSION = 2; // NOT a coordinate frame, indicates a mission command.
+      MAV_FRAME_GLOBAL_RELATIVE_ALT = 3; // Global (WGS84) coordinate frame + altitude relative to the home position.
+      MAV_FRAME_LOCAL_ENU = 4; // Local coordinate frame, Z-up (x: east, y: north, z: up).
+      MAV_FRAME_GLOBAL_INT = 5; // Global (WGS84) coordinate frame (scaled) + MSL altitude.
+      MAV_FRAME_GLOBAL_RELATIVE_ALT_INT = 6; // Global (WGS84) coordinate frame (scaled) + altitude relative to the home position.
+      MAV_FRAME_LOCAL_OFFSET_NED = 7; // Offset to the current local frame. Anything expressed in this frame should be added to the current local frame position.
+      MAV_FRAME_GLOBAL_TERRAIN_ALT = 10; // Global (WGS84) coordinate frame with AGL altitude (at the waypoint coordinate).
+      MAV_FRAME_GLOBAL_TERRAIN_ALT_INT = 11; // Global (WGS84) coordinate frame (scaled) with AGL altitude (at the waypoint coordinate).
+      MAV_FRAME_BODY_FRD = 12; // Body fixed frame of reference, Z-down (x: forward, y: right, z: down). Replacement for MAV_FRAME_BODY_NED, MAV_FRAME_BODY_OFFSET_NED.
+      MAV_FRAME_BODY_FLU = 13; // Body fixed frame of reference, Z-up (x: forward, y: left, z: up).
+      MAV_FRAME_LOCAL_FRD = 20; // Forward, Right, Down coordinate frame. This is a local frame with Z-down and arbitrary F/R alignment (i.e. not aligned with NED/earth frame). Replacement for MAV_FRAME_MOCAP_NED, MAV_FRAME_VISION_NED, MAV_FRAME_ESTIM_NED.
+      MAV_FRAME_LOCAL_FLU = 21; // Forward, Left, Up coordinate frame. This is a local frame with Z-up and arbitrary F/L alignment (i.e. not aligned with ENU/earth frame). Replacement for MAV_FRAME_MOCAP_ENU, MAV_FRAME_VISION_ENU, MAV_FRAME_ESTIM_ENU.
```

Comment:
Which frames do we actually need? Can we reduce it to e.g. 3? And which ones are actually supported by PX4?

Reply:
I founded thar these types not used in PX4 FW currently:
- MAV_FRAME_LOCAL_ENU
- MAV_FRAME_LOCAL_OFFSET_NED (PX4/Firmware#5407 related)
- MAV_FRAME_GLOBAL_TERRAIN_ALT
- MAV_FRAME_GLOBAL_TERRAIN_ALT_INT
- MAV_FRAME_LOCAL_FRD
- MAV_FRAME_LOCAL_FLU

I think most of it we can exclude from proto (except `MAV_FRAME_LOCAL_OFFSET_NED`, may be)